requirements.txt

```diff
@@ -20,3 +21,4 @@ slacker==0.13.0                  # library for sending slack messages
 slugify==0.0.1                   # used for encoding names for guids
 tqdm==4.40.2                     # convenient way to create progress bar for long-running command-line operations
 whitenoise==3.3.0                # simplified static file handling. Behind a major version due to missing Python 2 support
+csv23==0.3.2                     # provides the unicode-based API of the Python 3 csv module for Python 2 and 3.
```

Comment:
`seqr/views/utils/file_utils.parse_file` uses `csv.read()` to process the csv stream. When there are unicodes in the stream, it is not possible to have a Python 2 and 3 compatible code. The csv23 package solves this issue.